File: Mastermind.Education.Services/ApplicationCore/Entities/Enrollment.cs
using Mastermind.Education.Services.ApplicationCore.Entities;

namespace Mastermind.Education.Services.Entities
{
    public class Enrollment : BaseEntity
    {
        public int CourseId { get; set; }
        public int StudentId { get; set; }
        public string Grade { get; set; }

        public Course Course { get; set; }
        public Student Student { get; set; }
    }
}

File: Mastermind.Education.Web/ClientApp/src/app/models/enrollment.ts
export interface Enrollment {
  id: number;
  name: string;
  courseId: number,
  course: string,
  noOfDays: number,
  studentId: number,
  student: string,
  grade: string

}


export interface SaveEnrollment {
  id: number;
  courseId: number,
  noOfDays: number,
  studentId: number,
  grade: string

}
